spec/rubocop/cop/lint/shared_mutable_default_spec.rb

```diff
@@ -38,6 +38,14 @@
     end
   end
 
+  context 'when `capacity` keyword argument is used' do
```

Comment:
Might be a good idea to also have a test for a combination of a default value and `capacity:`.

Reply:
I have added a test case for `Hash.new(capacity:)`. However, this represents hash value omission introduced in Ruby 3.1 rather than a default value. Please let me know if this differs from the intended meaning of the comment.

Reply:
That's a good test, but I was thinking of something like:

`Hash.new({}, capacity: 42)` that would flag an offense. Basically a test for a combination of a default value and keyword params. (looking at the API those can be combined)